### PR TITLE
feat: add test for DropdownCardSwitch

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#71812

### CODE REVIEW

File: packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    wrapper.find(DropdownCard).first().simulate('press');
+
+    expect(
+      wrapper
+        .find(DropdownCard)
+        .first()
+        .contains(dropdownItems[0].childrenComp),
+    ).toBe(true);
+  });
```
Comment (vhu-axelor) [inline packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js:67]: Make another test to verify that children is not shown when car is closed

File: packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import {shallow} from 'enzyme';
+import {DropdownCardSwitch, DropdownCard, Text} from '@axelor/aos-mobile-ui';
+import {getGlobalStyles} from '../../tools';
+
+describe('DropdownCardSwitch', () => {
+  const dropdownItems = [
+    {key: 1, title: 'Item 1', childrenComp: <Text>Content 1</Text>},
+    {key: 2, title: 'Item 2', childrenComp: <Text>Content 2</Text>},
+  ];
+
```
Comment (vhu-axelor) [inline packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js:30]: Add test 'should render without crashing'

File: packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    expect(wrapper.find(DropdownCard).first().props().dropdownIsOpen).toBe(
+      true,
+    );
+
+    wrapper.find(DropdownCard).first().simulate('press');
+    expect(wrapper.find(DropdownCard).first().props().dropdownIsOpen).toBe(
+      false,
+    );
+  });
+
```
Comment (vhu-axelor) [inline packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js:53]: Add a test to verify that the title is passed to DropdownCard component

File: packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    expect(wrapper.find(DropdownCard).first().prop('dropdownIsOpen')).toBe(
+      true,
+    );
+
+    wrapper.find(DropdownCard).first().simulate('press');
+
+    expect(wrapper.find(DropdownCard).first().prop('dropdownIsOpen')).toBe(
+      false,
+    );
+  });
```
Comment (lme-axelor) [inline packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js:82]: That's two times the same test. 
Tranform into one test that check that card is opening when you press it and that it closes the others at the same time. There can be only one opened card at a time.

File: packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  it('renders correctly with dropdown items', () => {
+    const wrapper = shallow(
+      <DropdownCardSwitch dropdownItems={dropdownItems} />,
+    );
+    expect(wrapper.find(DropdownCard)).toHaveLength(dropdownItems.length);
+  });
```
Comment (lme-axelor) [inline packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js:44]: At space and check that props are correctly given to each DropdownCard (title, children)
(check StarScore, RadioSelect or MultiValuePicker for examples)

File: packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+  it('passes the correct title DropdownCard', () => {
+    const titleProps = 'Test';
+    const wrapper = shallow(
+      <DropdownCardSwitch dropdownItems={dropdownItems} title={titleProps} />,
+    );
+    dropdownItems.forEach((item, index) => {
+      const dropdownCard = wrapper.find(DropdownCard).at(index);
+      expect(dropdownCard.prop('title')).toEqual(item.title);
+    });
```
Comment (lme-axelor) [inline packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js:115]: Should go to 'renders correctly with dropdown items'
titleProps is useless here, DropdownCardSwitch does not have a title props.

**Pull request merged**